services/app/src/components/gameLayout/GameLayout.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import './gameLayout.css';
 
 const GameLayout = ({ children }) => (
-  <div className='game-view margin-t-m margin-auto'>
+  <div className='game-view margin-t-m margin-b-s margin-auto gap-sm'>
```

Comment:
check if margin-auto is still required. Maybe we can replace it with justify-content: center